data/tuhamworld.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Tunde Sanusi",
+  "bio": "I am a Frontend Engineer",
+  "avatar": "https://github.com/tuhamworld.png",
+  "links": [
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://linkedin.com/in/tuhamworld",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/tuhamworld",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Hashnode",
+      "url": "https://tuhamworld.hashnode.dev",
+      "icon": "SiHashnode"
+    },
```

```suggestion
    }
```
The trailing comma isnot required